cosmos_sdk_proto/src/prost/cosmos.staking.v1beta1.rs

```diff
@@ -1,1281 +0,0 @@
-    /// description defines the description terms for the validator.
-    #[prost(message, optional, tag="7")]
-    pub description: ::core::option::Option<Description>,
-    /// unbonding_height defines, if unbonding, the height at which this validator has begun unbonding.
-    #[prost(int64, tag="8")]
-    pub unbonding_height: i64,
-    /// unbonding_time defines, if unbonding, the min time for the validator to complete unbonding.
-    #[prost(message, optional, tag="9")]
-    pub unbonding_time: ::core::option::Option<::prost_types::Timestamp>,
-    /// commission defines the commission parameters.
-    #[prost(message, optional, tag="10")]
-    pub commission: ::core::option::Option<Commission>,
-    /// min_self_delegation is the validator's self declared minimum self delegation.
-    #[prost(string, tag="11")]
-    pub min_self_delegation: ::prost::alloc::string::String,
-}
-/// ValAddresses defines a repeated set of validator addresses.
-#[derive(Clone, PartialEq, ::prost::Message)]
-pub struct ValAddresses {
-    #[prost(string, repeated, tag="1")]
-    pub addresses: ::prost::alloc::vec::Vec<::prost::alloc::string::String>,
-}
-/// DVPair is struct that just has a delegator-validator pair with no other data.
-/// It is intended to be used as a marshalable pointer. For example, a DVPair can
-/// be used to construct the key to getting an UnbondingDelegation from state.
-#[derive(Clone, PartialEq, ::prost::Message)]
-pub struct DvPair {
-    #[prost(string, tag="1")]
-    pub delegator_address: ::prost::alloc::string::String,
-    #[prost(string, tag="2")]
-    pub validator_address: ::prost::alloc::string::String,
-}
-/// DVPairs defines an array of DVPair objects.
-#[derive(Clone, PartialEq, ::prost::Message)]
-pub struct DvPairs {
-    #[prost(message, repeated, tag="1")]
-    pub pairs: ::prost::alloc::vec::Vec<DvPair>,
-}
-/// DVVTriplet is struct that just has a delegator-validator-validator triplet
-/// with no other data. It is intended to be used as a marshalable pointer. For
-/// example, a DVVTriplet can be used to construct the key to getting a
-/// Redelegation from state.
-#[derive(Clone, PartialEq, ::prost::Message)]
-pub struct DvvTriplet {
-    #[prost(string, tag="1")]
-    pub delegator_address: ::prost::alloc::string::String,
-    #[prost(string, tag="2")]
-    pub validator_src_address: ::prost::alloc::string::String,
-    #[prost(string, tag="3")]
-    pub validator_dst_address: ::prost::alloc::string::String,
-}
-/// DVVTriplets defines an array of DVVTriplet objects.
-#[derive(Clone, PartialEq, ::prost::Message)]
-pub struct DvvTriplets {
-    #[prost(message, repeated, tag="1")]
-    pub triplets: ::prost::alloc::vec::Vec<DvvTriplet>,
-}
-/// Delegation represents the bond with tokens held by an account. It is
-/// owned by one delegator, and is associated with the voting power of one
-/// validator.
-#[derive(Clone, PartialEq, ::prost::Message)]
-pub struct Delegation {
```

Comment:
Staking and Distribution are not equivalent, for example this struct does not exist in  cosmos.distribution.v1beta1.rs